### PR TITLE
Trim Microsoft.AspNetCore

### PR DESCRIPTION
**Pull request by JamesNK**

### CODE REVIEW

**Comment by JamesNK** [comment]
Review please 🙏

File: src/DefaultBuilder/src/ConfigureWebHostBuilder.cs [hunk 1]
```diff
@@ -173,7 +173,7 @@ IWebHostBuilder ISupportsStartup.Configure(Action<WebHostBuilderContext, IApplic
         throw new NotSupportedException("Configure() is not supported by WebApplicationBuilder.WebHost. Use the WebApplication returned by WebApplicationBuilder.Build() instead.");
     }
 
-    IWebHostBuilder ISupportsStartup.UseStartup(Type startupType)
+    IWebHostBuilder ISupportsStartup.UseStartup([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.PublicMethods)] Type startupType)
```
Comment (davidfowl) [inline src/DefaultBuilder/src/ConfigureWebHostBuilder.cs:176]: Make this a constant?

Comment (halter73) [reply src/DefaultBuilder/src/ConfigureWebHostBuilder.cs:176]: I'd go the other way and just delete `StartupLinkerOptions`. This is simple enough I like just seeing the flags defined inline.

**Review by halter73 — APPROVED** [review]

**Review by davidfowl — APPROVED** [review]

**Comment by AraHaan** [comment]
So, what all gets trimmed with this?

**Pull request merged**